po/nb.po

```diff
@@ -15336,9 +15333,8 @@ msgid "<b>Quote Source Information</b>"
 msgstr "<b>Informasjon om noteringskilde</b>"
 
 #: gnucash/gtkbuilder/dialog-commodity.glade:238
-#, fuzzy
 msgid "<b>Security Information</b>"
-msgstr "Verdipapirinformasjon"
+msgstr "Sikkerhetsinfo</b>"
```

Comment:
- Verdipapir seems closer than Sikkerhet
- starting "<b>" missing

Reply:
Consistent even. I can add some context there if you make https://hosted.weblate.org/user/kingu/#managed an admin.

Reply:
> I can add some context there if you make https://hosted.weblate.org/user/kingu/#managed an admin.

Granted to help with the configuration. And you have become the revisor for nb.